### PR TITLE
Support for Goals:

### PR DESCRIPTION
**Pull request by bissont**
* List goals
* Delete goal

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,6 +41,15 @@ profile.followees().then(followees => {
 // Can lookup 'activityId' from 'currentActivityId' of profile() response
 profile.giveRideOn(otherRiderId, activityId);
 
+// Retrieve the goals you have
+goals = profile.getGoals();
+goals.then(goal=> {
+    console.log(goal); //JSON array of goals
+})
```
Comment (Ogadai) [inline README.md]: To keep the style similar for the above, I'd write this as:
````
profile.goals().then(goals => {
    console.log(goals); //JSON array of goals
})
````

File: src/Profile.js [hunk 2]
```diff
@@ -141,6 +141,14 @@ class Profile {
     }, 'application/json', 'json');
   }
 
+  getGoals() {
```
Comment (Ogadai) [inline src/Profile.js]: Keeping the naming convention for lists as above, can this just be `goals()` please?

**Review by Ogadai — CHANGES_REQUESTED** [review]
Thanks for the contribution @bissont. It looks great!

I'd prefer a slight tweak to the function name. Are you able to do that and update the PR please? I can do it, but I am busy today, so it will likely take me a few days to get to it.

File: README.md (hunk 1 above)
Comment (bissont) [reply README.md]: Done

**Review by Ogadai — APPROVED** [review]
Thanks @bissont

**Comment by Ogadai** [comment]
I've published v0.3.7 with these changes.

**Pull request merged**